bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -156,8 +156,7 @@ private synchronized static ResourceLimiter initializeResourceLimiter(BigtableOp
 
   private Watchdog watchdog;
   private final BigtableDataClient dataClient;
-
-  private final IBigtableDataClient clientWrapper;
+  private final RequestContext dataRequestContext;
```

Comment:
Why remove the wrapper? I thought the eventual goal is to migrate to using that interface only

Reply:
I removed the variable in `BigtableSession`.  Creating new `IBigtableDataClient` is cheap, and I added a new throttled `IBigtableDataClient`.